marlowe-playground-client/src/Marlowe/Parser.purs

```diff
@@ -15,6 +15,7 @@ import Data.List (List)
 import Data.Maybe (Maybe(..))
 import Data.String (length)
 import Data.String.CodeUnits (fromCharArray)
+import Data.Unit
```

Comment:
should be explicit

Reply:
@shmish111 Data.Unit has only one type and one function defined in it: http://hackage.haskell.org/package/unit-0.1.0.1/docs/Data-Unit.html

If import them explicitly - I'm still importing the whole module :)
I can do that just in case though.

Reply:
please do, it's the rule we use and for good reason, it can be difficult to read when you don't know where something is coming from. Even for something like `Data.Unit` it makes sense because it hardly adds any overhead but maintains consistency. Also, `psc-ide` will add it as an explicit import so you shouldn't need to ever do it yourself anyway.